n6001/hardware/ofs_n6001/build/scripts/run.sh

```diff
@@ -15,6 +15,11 @@ echo "run.sh KERNEL_BUILD_PWD is $KERNEL_BUILD_PWD"
 BSP_BUILD_PWD="$KERNEL_BUILD_PWD/../"
 echo "run.sh BSP_BUILD_PWD is $BSP_BUILD_PWD"
 
+Q_REVISION="ofs_top"
```

Comment:
we should get these from the db file that came along with the FIM's pr-build-template, but this is ok for now.